syntax = "proto3";

message HelloRequest {
  string name = 1;
}

message HelloReply {
  string message = 1;
}

service Commands {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}